src/app/feature-flag/toggles.module.ts

```diff
@@ -0,0 +1,18 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { Http } from '@angular/http';
+
+import { FeatureFlagModule } from 'ngx-feature-flag';
+import { TogglesRoutingModule } from './toggles-routing.module';
+
+
+@NgModule({
+  imports: [
+    CommonModule,
+    FeatureFlagModule,
+    TogglesRoutingModule
+  ]
+})
+export class TogglesModule {
+  constructor(http: Http) {}
```

Comment:
Why is `Http` in the constructor of a module?

Reply:
because of bad copy/paste :))
but even worse notice this http arg is in most module define \0/
let me fix...